components/add_groups_to_team_modal/add_groups_to_team_modal.test.jsx

```diff
@@ -83,9 +86,9 @@ describe('components/AddGroupsToTeamModal', () => {
         setTimeout(() => {
             expect(instance.handleResponse).toBeCalledTimes(2);
         }, 0);
```

Comment:
Can we refactor this to the following? Not sure why it's written the way it is
```suggestion
        expect(instance.handleResponse).toBeCalledTimes(2);
```